app/services/data_migrations/backfill_training_locations_on_candidate_preferences.rb

```diff
@@ -0,0 +1,22 @@
+module DataMigrations
+  class BackfillTrainingLocationsOnCandidatePreferences
+    TIMESTAMP = 20250609145824
+    MANUAL_RUN = false
```

Comment:
The migration will alter about 8,750 records (or thereabouts). I do not believe this will have any significant impact on database performance, but if we are worried about it, we can always change it to be run manually and I can run it early Wednesday morning.